plugins/mana/plugin.go

```diff
@@ -47,6 +47,8 @@ var (
 	consensusBaseManaPastVectorStorage         *objectstorage.ObjectStorage
 	consensusBaseManaPastVectorMetadataStorage *objectstorage.ObjectStorage
 	consensusEventsLogStorage                  *objectstorage.ObjectStorage
+	consensusEventsLogsStorageSize             int64
+	consensusEventsLogsStorageSizeMu           sync.Mutex
```

Comment:
an atomic counter does the job, no need for mutex. Consider the `go.uber.org/atomic` package or the built in `sync/atomic` https://gobyexample.com/atomic-counters

Reply:
also, probably we don't need `int64`, an `uint32` might be more than enough with couple hundred thousand max storage size.

Reply:
Totally forgot about atomic.

Reply:
>also, probably we don't need int64, an uint32 might be more than enough with couple hundred thousand max storage size.

This is one of those things that just come down to preference. I don't see any real benefit of `uint` here.  If we were doing stuff like with bit shifts or overflow maths, it'll be useful. Furthermore, it impaires readability with all the castings here and there. 
A `length, size, capacity, index` should be an `int`.  Whether you want `int32` or `int64` depends on memory or intent.